_pytest/trainers/test_mitie_trainer.py

```diff
@@ -7,4 +7,4 @@ class TestMitieTrainer:
     def test_failure_on_invalid_lang(self):
         config = RasaNLUConfig("config_mitie.json")
         with pytest.raises(NotImplementedError):
-            MITIETrainer(config.mitie_file, 'umpalumpa')
+            MITIETrainer(config['mitie_file'], 'umpalumpa')
```

Comment:
yes `.mitie_file` works as well.

Nevertheless, the dot notation raises a warning in IDEs `mitie_file might not be defined`, whereas the other notation makes clear that it is some sort of dictionary (where values might not be defined)